tox.ini

```diff
@@ -9,7 +10,18 @@ deps =
 commands =
 	py.test {posargs}
 
+[testenv:negtz]
+passenv = LANG
+setenv =
+    TZ = America/Argentina/Buenos_Aires
+deps =
```

Comment:
These seems to be a bug in `tox` that prevents us from using this 100%:

`commands = {[testenv]commands}`

Does not substitute `{posargs}` in `commands` before the inheritance :(